CODE_OF_CONDUCT.md

```diff
@@ -2,22 +2,26 @@
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our
+project and our community a harassment-free experience for everyone.
 
 ## Our Standards
 
 Examples of behavior that contributes to creating a positive environment include:
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+* Not being disrespectful of differing viewpoints and experiences
+(i.e., keep your mouth shut if you can only think of something negative
+and useless to say).
+
+* Accept or ignore constructive criticism. The choice is yours. But
+don't attack someone who has given you a respectful opinion of your
+code or doc changes.
 
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
+* Trolling, insulting/derogatory comments, and personal, political, or religious attacks
```

Comment:
The oxford comma before and here is actually misplaced.

* Trolling, insulting/derogatory comments and personal, political or religious attacks